Sources/Crypto/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# This source file is part of the SwiftCrypto open source project
+#
+# Copyright (c) 2021 Apple Inc. and the SwiftCrypto project authors
+# Licensed under Apache License v2.0
+#
+# See LICENSE.txt for license information
+# See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(Crypto
```

Comment:
Should this be a static library or a shared library?  At least on Windows, static linking doesn't work yet, and needs to be shared.

Reply:
You mean this should say `add_library(Crypto SHARED`? The examples that I've looked at don't specify `SHARED` and only the C modules have `STATIC`.

Reply:
Well, the current signature leaves it in the control of the user.  If you specify `-D BUILD_SHARED_LIBS=YES` then this is implicitly a dynamic library and otherwise (or if you do `-D BUILD_SHARED_LIBS=NO`) it is implicitly a static library.  The reason for not specifying an explicit shared/static here is to allow the user to select the build type.  This is why setting the default value for `BUILD_SHARED_LIBS` to `YES` is what I suggested above.